implementations/rust/ockam/ockam_command/src/util/api.rs

```diff
@@ -87,6 +87,7 @@ pub(crate) fn create_secure_channel(
         addr,
         authorized_identifiers,
         credential_exchange_mode,
+        None,
```

Comment:
Can you check if this function is used elsewhere? If not, you could delete it and build the request in the command directly.

Also, IIUC, I think the new argument will have no effect if you are passing `None` here.

Reply:
Sorry didn't get that, did you mean `create_secure_channel` ? if yes, then yes it is used in two different locations in `ockam_command/src/project/util.rs` [here](https://github.com/build-trust/ockam/blob/2de6faaac4bc1ee7c7dc06dc914dd7408be11de8/implementations/rust/ockam/ockam_command/src/project/util.rs#L111) and [here](https://github.com/build-trust/ockam/blob/2de6faaac4bc1ee7c7dc06dc914dd7408be11de8/implementations/rust/ockam/ockam_command/src/project/util.rs#L111)

Reply:
Yeah, I meant `create_secure_channel` sorry. Now that I see the changes again, let us remove that method and build the request just as you do in the `secure_channel create` command. 

Without the context of this PR, anyone reading the `create_secure_channel` method contents will find it confusing trying to figure out why we are passing None as the identity.